source/_docs/installation/virtualenv.markdown

```diff
@@ -14,10 +14,8 @@ If you already have Python 3.6 or later installed (we suggest 3.7 or later), you
 
 It's recommended when installing Python packages that you use a [virtual environment](https://docs.python.org/3.6/library/venv.html#module-venv). This will make sure that your Python installation and Home Assistant installation won't impact one another. The following steps will work on most UNIX like systems.
 
-_(If you're on a Debian based system, you will need to install Python virtual environment support using `apt-get install python3-pip python3-venv`. You may also need to install development libraries using `apt-get install build-essential libssl-dev libffi-dev python3-dev`.)_
-
-<p class='Note'>
-It is recommended to use the [advanced guide](/docs/installation/raspberry-pi/) which allows for the installation to run as a `homeassistant` user. The steps below may be shorter but some users find difficulty when applying updates and may run into issues.
+<p class='note'>
+This is a generic guide for running Home Assistant under Python. We recommend to use [our recommended installation guides](/docs/installation/#recommended) or use one of the community contributed guides for [your device or operationg system](/docs/installation/#community-provided-guides). The steps below may be shorter but some users find difficulty when applying updates and may run into issues.
```

Comment:
Not sure we should be recommending the community guides, which is how that sounds. Maybe

We recommend to use [our recommended installation guides](/docs/installation/#recommended).  Alternatively you can use one of the community contributed guides for [your device or operating system](/docs/installation/#community-provided-guides).

Reply:
I really want to change the navigation and general pages around the installation methods first (especially around the community guidelines part tbh)

Reply:
With changing it first, would you prefer me not add any changes to this paragraph for now ?